### DISCUSSION TITLE
API PUBLIC 404 Not Found

### DISCUSSION THREAD

**Discussion by Javi-MC** [globaleaks/globaleaks-whistleblowing-software#3545] [Q&A]
### What version of GlobaLeaks are you using?

4.11.5

### What browser(s) are you seeing the problem on?

All

### What operating system(s) are you seeing the problem on?

Linux

### Describe the issue

Hello everyone! I wanted to ask because I am trying to create Globaleaks 4.11.5 on Ubuntu 22.04, with a certificate for SSL connection and when I use Tor with the onion address everything works fine, but when I try to use the domain with https from any browser (Tor included) I always get the 404 error /api/public not found, with the onion address I can see the JSON in api/public.

And no matter how hard I try to find out where the error comes from, I can't find the solution, to see if someone could give me a hand with this.

Thank you so much!!

**Comment by evilaliv3** [comment]
Hello @Javi-MC 

Thank you for your question; do you a have a public link where i could test this?

**Comment by Javi-MC** [comment]
Hi @evilaliv3, the site is  
https://denuncias.miguelturra.es/#/

**Comment by evilaliv3** [comment]
Thank you @Javi-MC 

Actually the problem is in your proxy implementation.

Let me point out that globaleaks shall never be used with a proxy because a proxy as significant impact on the overall security of the platform.

**Reply by Javi-MC** [reply]
Thanks @evilaliv3 , I have not configured any proxy option, how and where can I correct it?

Thanks!!

**Reply by Javi-MC** [reply]
Hi again @evilaliv3 I'm still stuck with this issue, could you help me?  Thanks in advance!!

**Comment by Javi-MC** [comment]
In the end my problem was that I was configuring the virtual hosts and the certificate in the apache config in Ubuntu. After clearing all the settings and using only the GlobaLeaks settings it works fine.
Greetings!!

**Reply by Marcbcn23** [reply]
Hola Javi, 

Tengo el mismo problema con mi web. Por lo que entiendo, tuviste que desactivar Apache para que funcionara? Y no necesitas Apache para que haga funcionar php y mysql? Donde quitaste esos ajustes? Gracias por adelantado!

**Reply by Javi-MC** [reply]
Sí, eliminé todos los "virtual host" que había creado en la carpeta de Apache y configuré todo desde las propias opciones de globaleaks, incluido el certificado SSL.
